frontend/src/global_styles/content/_projects_list.sass

```diff
@@ -167,10 +167,17 @@ form.project-filters
 // just as in the work package table.
 body.controller-projects.action-index
   #content
-    overflow: visible
```

Comment:
@bsatarnejad As discussed, we should always be a bit carful about changes on the bigger containers like "content" or some specific solutions for single pages. So the better way would always be a more generic solution like a modifier. I reversed the changes here and added a generic container instead.

Reply:
Hi @ingamai . Thank you so much for your help. It is good to calculate the height of content but the problem we talked about still exists, after i click on three dots icon to open the additional menu on the last row of the table, it is opened in the bottom of the page.